iphone/Classes/PlatformModule.m

```diff
@@ -220,28 +221,38 @@ - (NSString *)identifierForVendor
 }
 
 #if defined(USE_TI_PLATFORMIDENTIFIERFORADVERTISING) || defined(USE_TI_PLATFORMGETIDENTIFIERFORADVERTISING)
-- (NSNumber *)isAdvertisingTrackingEnabled
+- (BOOL)isAdvertisingTrackingEnabled
 {
-  return NUMBOOL([[ASIdentifierManager sharedManager] isAdvertisingTrackingEnabled]);
+  return [[ASIdentifierManager sharedManager] isAdvertisingTrackingEnabled];
 }
 
 - (NSString *)identifierForAdvertising
 {
   return [[[ASIdentifierManager sharedManager] advertisingIdentifier] UUIDString];
 }
+#else
+- (BOOL)isAdvertisingTrackingEnabled
+{
+    return NO;
+}
+
+- (NSString *)identifierForAdvertising
+{
+    return @"";
+}
 #endif
 
 - (id)id
 {
   return [TiUtils appIdentifier];
 }
 
-- (NSString *)createUUID:(id)args
+- (NSString *)createUUID
```

Comment:
Another one: Is our proxy now able to handle methods without the unused args? That'd be lit! But we would also need backwards compatibility to not break literally ALL available modules.

Reply:
As in "Do I still need to define arguments for a method even if we don't accept/use any?" Then no, you do not need to add useless arguments now. There's an example right in their very sparse docs here for a description method with no args: https://developer.apple.com/documentation/javascriptcore/jsexport?language=objc